src/types/options/statisticsRequestOptions.ts

```diff
@@ -28,7 +29,8 @@ export interface StatisticsRequestOptionsSources extends StatisticsRequestOption
 
 export interface StatisticsGeometryRequestOptions {
   requestTimeout?: number
-  statisticsGroup: StatisticsGroupId
+  statisticCollectionId?: number
+  statisticsGroup?: StatisticsGroupId
```

Comment:
statisticsGroupId ?

Reply:
no its correct. was always called like that